pkg/pocket/pkg/logger/logger.go

```diff
@@ -56,7 +58,7 @@ func (l *Logger) writeLine(line string) error {
 	if l.mute {
 		return nil
 	} else if l.print {
-		log.Info(line)
+		log.Info(fmt.Sprintf("[%s] %s", l.name, line))
```

Comment:
When the log is printed to the stdout, I use the name filed to specify which client the log is belong to.